### ISSUE TITLE
Unable to establish a connection to communicate with the server's self-compiled version of ENET1.3.17.

### ISSUE THREAD

**Issue by leizhiren** [JohannesDeml/ENetUnityMobile#10]
与服务器自行编译的ENET1.3.17版本无法建立连接进行通信。

So I'm using this framework called Skynet 1.6.0 (it's written in Lua 5.4.4) on the server side. We compiled our own version of ENET 1.3.17 as well.
We did some cross-machine testing on the server and everything was working fine, but when I tried using the libraries from the Release version, I couldn't connect. But your ENerServer in the project works just fine. I'm at a loss here, can you help me out?

服务端使用的是Skynet1.6.0框架（Lua5.4.4）。服务端自行编译的ENET1.3.17版本。

服务器自行跨机器测试连接通信是正常的，但是我使用Release中的库进行连接就连接不上，但是你项目中的ENerServer启动起来却是能正常连接的。现在我不知道怎么办？请求帮助。

**Comment by JohannesDeml** [comment]
Hi @leizhiren,

Sadly I don't have any experience with Skynet and the original ENet package, but just with the C# wrapper from https://github.com/nxrighthere/ENet-CSharp
This is also the package that is used here (in version 2.4.8, I guess this is a different versioning than from the original package).

**Comment by leizhiren** [comment]
好吧，那是挺遗憾的。那你能帮助我对原始enet进行编译让其能在Unity中使用吗？我编译出来的库似乎在Unity无法运行，应该是与C#层面不对应。原始Enet的仓库在：https://github.com/lsalzman/enet。

Well, that's too bad. Can you help me compile the original enet to make it work in Unity? The libraries I compiled don't seem to work in Unity. It's probably not compatible with the C# layer. The original enet repository is located at  https://github.com/lsalzman/enet

**Comment by JohannesDeml** [comment]
No, sorry, as I said, I don't have any experience with the library from Lee Salzman, and I also don't think it is an easy task to pull off given that there is this wrapper (that this project uses), that I didn't write, but that took quite some time to develop.
